example-get-started-cv/code/notebooks/TrainSegModel.ipynb

```diff
@@ -154,7 +172,8 @@
     "    live = Live(dir=os.path.join('results', 'train'), \n",
     "            report=\"md\", \n",
     "            save_dvc_exp=True)\n",
-    "    live.log_param(\"base_lr\", base_lr)\n",
+    "    live.summary[\"base_lr\"] =base_lr\n",
+    "    live.make_summary()\n",
```

Comment:
@alex000kim any reason to not make this a parameter?

Reply:
Do you mean via `.log_param()`? 
I can update that, but for this one, I simply followed the docs examples for fastai:
https://dvc.org/doc/dvclive/api-reference/ml-frameworks/fastai#examples 
<img width="857" alt="image" src="https://user-images.githubusercontent.com/17241029/215537524-325fbf15-852a-4d2e-853e-97c46d5c0057.png">